notes/05_infomax/1_primer_info_theory.tex

```diff
@@ -446,7 +446,7 @@ \subsection{Relationship between the KL-Divergence and Mutual Information}
 D_{KL} \lbrack \, {\color{red}p_X(x) p_Y(y)} \, || \, {\color{blue}p_{X,Y}(x,y)}\, \rbrack
 \end{equation}
 
-The above is only equal if both of them are equal to zero, which is satisfied iff $p_{X,Y}(x,y) = p_X(x) p_Y(y)$.
```

Comment:
not a typo, iff stands for "if and only if". could make it bold

Reply:
ok gotcha! didn't know that one, thanks

Reply:
and I think it's fine w/o bold, makes sense now